typescript-selenium/pageobjects/dashboard/NewWorkspace.ts

```diff
@@ -0,0 +1,189 @@
+/*********************************************************************
+ * Copyright (c) 2019 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+import { injectable, inject } from "inversify";
+import { DriverHelper } from "../../utils/DriverHelper";
+import { CLASSES } from "../../inversify.types";
+import { TestConstants } from "../../TestConstants";
+import { By } from "selenium-webdriver";
+import 'reflect-metadata';
+import { Dashboard } from "./Dashboard";
+import { Workspaces } from "./Workspaces";
+import { Ide } from "../ide/Ide";
+import { TestWorkspaceUtil, WorkspaceStatus } from "../../utils/workspace/TestWorkspaceUtil";
+import { WorkspaceDetails } from "./workspace-details/WorkspaceDetails";
+
+
+@injectable()
+export class NewWorkspace {
```

Comment:
What about changing the name to e.g. ``` WorkspaceHandler ``` and include the methods ``` stopWorkspaceByUI ``` and ``` deleteWorkspaceByUI ``` here too, so all the logic arount handling the workspace will be on once place? Or is there some reason to have it separated?

Reply:
The "NewWorkspace" pageobject describes "NewWorkspace" page